e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -69,8 +69,8 @@ describe('team management', () => {
 
               const systemTeams = ['admins', 'editors', 'viewers'];
               systemTeams.forEach(name => {
-              cy.get('#table-container chef-tr').contains(name)
-                .parent().parent().find('chef-control-menu').as('control-menu');
+                cy.get('#table-container chef-tr').contains(name)
```

Comment:
used an auto-formatter on this file which caused a lot of little changes like this